api/app/routers/hfi_calc.py

```diff
@@ -47,7 +46,8 @@ async def get_daily_view(response: Response,
 
     async with ClientSession() as session:
         header = await get_auth_header(session)
-        wfwx_stations = await get_wfwx_stations_from_station_codes(session, header, station_codes)
+        wfwx_stations = await app.wildfire_one.wfwx_api.get_wfwx_stations_from_station_codes(
```

Comment:
```suggestion
        wfwx_stations = await get_wfwx_stations_from_station_codes(
```

Reply:
Minor: could make this a bit cleaner by just importing `get_wfwx_stations_from_station_codes` from `wfwx_api` (as you're already doing with a bunch of other functions in that file)